builtins/web/url.cpp

```diff
@@ -187,8 +186,7 @@ jsurl::SpecSlice URLSearchParams::serialize(JSContext *cx, JS::HandleObject self
 
 bool URLSearchParams::append(JSContext *cx, unsigned argc, JS::Value *vp) {
   METHOD_HEADER(2)
-  bool err = false;
-  auto value = append_impl_validate(cx, args[0], &err, "append");
+  auto value = append_impl_validate(cx, args[0], "append");
```

Comment:
The URL iteration doesn't do key validation before value validation like the headers one does currently, although further investigation into the failing URL tests might indicate implementing this can resolve some remaining WPT there. As a headers PR I didn't want to get into that here though.